js/twitterFetcher.js

```diff
@@ -60,7 +60,7 @@
 
   function strip(data) {
     return data.replace(/<b[^>]*>(.*?)<\/b>/gi, function(a,s){return s;})
-        .replace(/class=".*?"|data-query-source=".*?"|dir=".*?"|rel=".*?"/gi,
+        .replace(/class="(?!(tco-hidden|tco-display|tco-ellipsis))+.*?"|data-query-source=".*?"|dir=".*?"|rel=".*?"/gi,
```

Comment:
So just to confirm as it is always fun looking at a regex a year after it was made... your addition here keeps the 3 classes you defined, but will still remove other classes that do not match right? This has been tested?